backend/app/api/crypto.py

```
@@ -0,0 +1,1377 @@
+                            )
+                        )
+                    )
+                    recent_blockchain_txs = blockchain_tx_count.scalar() or 0
+
+                    wallet_sync_status = {
+                        "wallet_configured": True,
+                        "wallet_address": portfolio.wallet_address,
+                        "recent_blockchain_transactions": recent_blockchain_txs,
+                        "last_sync_check": datetime.utcnow().isoformat()
+                    }
+                except Exception as e:
```

Ruff (0.14.0)

Do not catch blind exception: `Exception`

(BLE001)